sqlutil/open.go

```diff
@@ -15,7 +15,10 @@ import (
 )
 
 var (
-	defaultOptions = &builder{parser: sqlparser.DefaultSQLParser()}
+	defaultOptions = &builder{
+		parser:  sqlparser.DefaultSQLParser(),
+		options: []DBOption{WithMaxOpenConns(1 << 7)},
```

Comment:
```suggestion
		options: []DBOption{WithMaxOpenConns(128)},
```

Maybe write 128 explicitely?